Makefile

```diff
@@ -48,28 +48,28 @@ docker_compose_build:
 # dependency on generate needed to create generated file outside of docker with
 # current user owner instead of root
 docker_test: generate
-	$(DOCKER_COMPOSE) up --build --abort-on-container-exit --exit-code-from tigris_test tigris_test
+	$(DOCKER_COMPOSE) up --build tigris_test tigris_test
 
 docker_test_no_build:
-	$(DOCKER_COMPOSE) up --no-build --abort-on-container-exit --exit-code-from tigris_test tigris_test
+	$(DOCKER_COMPOSE) up --no-build tigris_test tigris_test
 
-test: clean docker_test
+test: docker_test
```

Comment:
I think it makes sense to avoid a clean each time we run these. Having said that, should we mention this behavior change in the README?